src/arrayMethodSort.js

```diff
@@ -4,8 +4,36 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const compare = (value1, value2) => {
+    const strValue1 = value1.toString();
+    const strValue2 = value2.toString();
+
+    if (strValue1 === strValue2) {
+      return 0;
+    }
+
+    return (strValue1 < strValue2) ? -1 : 1;
```

Comment:
redundant parentheses
```suggestion
    return strValue1 < strValue2 ? -1 : 1;
```

Reply:
> redundant parentheses

Я їх додала для кращої читабельності. Додавати дужки не треба, навіть якщо, на мій погляд, покращиться сприйняття коду?